bin/main.rb

```diff
@@ -1,2 +1,30 @@
-#!/usr/bin/env ruby
-puts 'Hello world!'
+require_relative '../lib/Player'
+require_relative '../lib/Game'
+
+puts 'Please enter player1 name: '
+player_1_name = gets.chomp
+player1 = Player.new(player_1_name, 'x')
+
+puts 'Please enter player2 name: '
+player_2_name = gets.chomp
+player2 = Player.new(player_2_name, 'o')
+
+game = Game.new(player1, player2)
+
+puts "Welcome to the Tic - Tac - Toe game #{player1.name} and #{player2.name}"
+game.board.print_board
+loop do
+  game.move(player1)
+  break if game.who_won != ''
+
+  puts ''
+  puts '- - - - - - - -'
+  game.move(player2)
```

Comment:
- [ ] **OPTIONAL** Try to be consistent, it looks weird that this line only appears in some turns. make it appear always or delete it 
![image](https://user-images.githubusercontent.com/53256139/91372501-68c6cd00-e7d9-11ea-9884-61fe25d5d2b1.png)

Reply:
Thank you, we did it.